vcd/resource_vcd_org_vdc.go

```diff
@@ -491,13 +499,13 @@ func resourceVcdVdcUpdate(d *schema.ResourceData, meta interface{}) error {
 		return fmt.Errorf("error updating VDC metadata: %s", err)
 	}
 
-	log.Printf("[TRACE] vdc update completed: %s", adminVdc.AdminVdc.Name)
+	log.Printf("[TRACE] VDC update completed: %s", adminVdc.AdminVdc.Name)
 	return resourceVcdVdcRead(d, meta)
 }
 
-// Deletes a vdc, optionally removing all objects in it as well
+// Deletes a VDC, optionally removing all objects in it as well
 func resourceVcdVdcDelete(d *schema.ResourceData, meta interface{}) error {
-	log.Printf("[TRACE] vdc delete started")
+	log.Printf("[TRACE] VDC delete started %s", d.Get("name").(string))
```

Comment:
Thanks for adding this. Trivial thing, I'd add a separator, so it's clearly visible where the VDC name starts:
```suggestion
	log.Printf("[TRACE] VDC delete started: %s", d.Get("name").(string))
```

Reply:
changed

Reply:
changed

Reply:
Thanks! After your changes mow all VDC CRUD operations have consistent logging with added detail:
```
log.Printf("[TRACE] VDC creation initiated: %s", orgVdcName)
log.Printf("[TRACE] VDC read initiated: %s", vdcName)
log.Printf("[TRACE] VDC update initiated: %s", vdcName)
log.Printf("[TRACE] VDC delete started: %s", vdcName)
```